### DISCUSSION TITLE
0.0.44

### DISCUSSION THREAD

**Discussion by shuding** [vercel/satori#289] [Announcements]
## What's Changed
* Fixes 234 background image url fails when using quotes by @Sam-Apostel in https://github.com/vercel/satori/pull/256
* Add support for passing in `0` as first argument to `linear-gradient` and `repeating-linear-gradient` + color model tests by @jueungrace in https://github.com/vercel/satori/pull/251
* Better kerning by @shuding in https://github.com/vercel/satori/pull/258
* Add docs about asm.js/WASM/native differences  by @mrkldshv in https://github.com/vercel/satori/pull/259
* chore: use lowercase for resvg-js names by @yisibl in https://github.com/vercel/satori/pull/279
* feat: throw explicit errors when useing unsupported image types by @LuciNyan in https://github.com/vercel/satori/pull/276
* Support multiple directions in linear-gradient by @shuding in https://github.com/vercel/satori/pull/284
* Fix image size resolution by @shuding in https://github.com/vercel/satori/pull/285
* Support Noto math and symbol fonts by @shuding in https://github.com/vercel/satori/pull/286

## New Contributors
* @jueungrace made their first contribution in https://github.com/vercel/satori/pull/251
* @yisibl made their first contribution in https://github.com/vercel/satori/pull/279

**Full Changelog**: https://github.com/vercel/satori/compare/0.0.43...0.0.44

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/satori/releases/tag/0.0.44'>0.0.44</a>.</em>

**Comment by joulev** [comment]
Hi, do you know when will `@vercel/og` ship with this?
